### DISCUSSION TITLE
Using `<script>` for non-external scipts with page_scripts?

### DISCUSSION THREAD

**Discussion by OndraM** [orestbida/cookieconsent#126] [Q&A]
Hi,
the `page_scripts` option is described as "Manage third party scripts" with this being an examlpe:

```html
<script type="text/plain" data-cookiecategory="analytics" src="analytics.js" defer></script>
```

however, I discovered it also works for non-external script statements like this:
```html
<script type="text/plain" data-cookiecategory="analytics">
    alert('consent given');
</script>
```

So is this an intentional but undocumented behavior? Should it work like this - can we use it without risk of being removed in the future?

Thanks!

**Comment by orestbida** [comment]
Yes, this definitely the intended behaviour! I called it "third party" because generally you'd use the plugin to load scripts like Google Analytics etc. 

Rest assured, I consider it a core feature and I don't have plans on removing/modifying it.

**Reply by OndraM** [reply]
Ok, thanks! Would be great ho have this as en example in the readme - I found this feature by coincidence :)
